js/util/script_detection.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const isVerticalWritingModeRegExp = new RegExp([
+    '[ᄀ-ᇿ]',
+    '[가-힣]',
+    '[ㄱ-ㆎ]',
+    '[一-鿌]',
+    '[㐀-䶵]',
+    // eslint-disable-next-line no-irregular-whitespace
+    '[　-〿]',
```

Comment:
Ah, you’re right. I was seeing a quirk in Gecko’s text selection behavior that causes the [Character Identifier](https://dbaron.org/mozilla/char-identifier/) extension to conflate various spaces. The escaped sequence would be less confusing in this case.